### PR TITLE
Dmp 1165

### PR DESCRIPTION
**Pull request by HMCTSDeveloper**

### CODE REVIEW

**Comment by HMCTSDeveloper** [comment]
**Orignal Hibernate Query**
   select
        *
    from
        darts.court_case c1_0 
    join
        darts.hearing h1_0 
            on c1_0.cas_id=h1_0.cas_id 
    join
        darts.courtroom c2_0 
            on c2_0.ctr_id=h1_0.ctr_id 
    join
        darts.courthouse c3_0 
            on c3_0.cth_id=c2_0.cth_id 
    where
        upper(c3_0.courthouse_name) like '%SWANSEA%'
        and upper(c2_0.courtroom_name) like '%1%'
	

**New Query** 
	 select
       *
    from
        darts.court_case c1_0 
    join
        darts.courthouse c2_0 
            on c2_0.cth_id=c1_0.cth_id 
    join
        darts.hearing h1_0 
            on c1_0.cas_id=h1_0.cas_id 
    join
        darts.courtroom c3_0 
            on c3_0.ctr_id=h1_0.ctr_id 
    where
        upper(c2_0.courthouse_name) like ? escape '' 
        and upper(c3_0.courtroom_name) like ? escape ''

File: src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java [hunk 1]
```diff
@@ -148,14 +149,15 @@ private List<Predicate> addJudgeCriteria(GetCasesSearchRequest request, Criteria
     private List<Predicate> addHearingDateCriteria(GetCasesSearchRequest request, CriteriaBuilder criteriaBuilder, Root<CourtCaseEntity> caseRoot) {
         List<Predicate> predicateList = new ArrayList<>();
         if (request.getDateFrom() != null || request.getDateTo() != null) {
-            Join<CourtCaseEntity, HearingEntity> hearingJoin = joinHearing(caseRoot);
             if (request.getDateFrom() != null) {
+                Join<CourtCaseEntity, HearingEntity> hearingJoin = joinHearing(caseRoot);
```
Comment (cmagowan-hmcts) [inline src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java:153]: did you mean to revert this change?

Comment (HMCTSDeveloper) [reply src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java:153]: ACCEPT.

**Review by cmagowan-hmcts — APPROVED** [review]
